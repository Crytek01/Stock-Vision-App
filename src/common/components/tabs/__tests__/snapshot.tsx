import { render } from "@testing-library/react";
import { TabPanel } from "../tab-panel.component";
it("renders tab panel component", () => {
  const { container } = render(<TabPanel index={0} value={0} />);
  expect(container).toMatchSnapshot();
});
