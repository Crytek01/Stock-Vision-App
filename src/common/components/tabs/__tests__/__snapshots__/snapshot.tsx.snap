// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders tab panel component 1`] = `
<div>
  <div
    aria-labelledby="simple-tab-0"
    id="simple-tabpanel-0"
    role="tabpanel"
  >
    <div
      class="MuiBox-root css-19kzrtu"
    />
  </div>
</div>
`;
